frontend/src/queries/nodes/InsightQuery/utils/queryNodeToFilter.ts

```diff
@@ -167,7 +179,7 @@ export const queryNodeToFilter = (query: InsightQueryNode): Partial<FilterType>
         Object.assign(filters, objectClean<Partial<Record<keyof CompareFilter, unknown>>>(query.compareFilter))
     }
 
-    if (!isStickinessQuery(query)) {
+    if (!isStickinessQuery(query) && 'aggregation_group_type_index' in query) {
```

Comment:
Note: I could add those fields to the Base web analytics query runner as well and leave them unused if you guys prefer it. That way we would not need the type guarding here